mlflow/utils/databricks_utils.py

```diff
@@ -103,12 +103,7 @@ def is_databricks_default_tracking_uri(tracking_uri):
 
 @_use_repl_context_if_available("isInNotebook")
 def is_in_databricks_notebook():
-    if _get_property_from_spark_context("spark.databricks.notebook.id") is not None:
-        return True
-    try:
-        return acl_path_of_acl_root().startswith("/workspace")
-    except Exception:
-        return False
+    return "DATABRICKS_RUNTIME_VERSION" in os.environ
```

Comment:
Does this work in older versions of runtime?

Reply:
Yes.
https://github.com/databricks/universe/blob/9188781c8a5563b0adf1bc7350a8edbdd023eab5/spark/images/build_image.bzl#L175

at least starting from 9.0, it has this.

Reply:
Yes.